### PR TITLE
Updating redirect url docs for core 2

### PR DESCRIPTION
**Pull request by jescalan**
Core 2 changed the way that redirect url props and env variables work and it requires a number of updates to the docs for this.

### CODE REVIEW

File: docs/account-portal/custom-redirects.mdx [hunk 1]
```diff
@@ -5,21 +5,35 @@ description: Customize where your users are redirected to after they sign in or
 
 # Customizing your Account Portal redirects
 
-After a user signs in or signs up, Clerk automatically redirects users back to your application by appending a `redirect_url` query param on the Account Portal page. However, Clerk provides various options for you to customize where your users are redirected to.
+When a user clicks on a link or button to navigate to a Clerk sign up or sign in page, Clerk will persist the page the user was on before into a `redirect_url` querystring, and navigate back to that page after the sign up or sign in is completed in order to not interrupt the user's flow through your app. So, for example, if you were on `example.com/foo`, and clicked a "sign in" button on that page, it might take you to `example.com/sign-in?redirect_url=example.com/foo`. After the sign in was complete, you'd be taken back to `example.com/foo`. However, Clerk provides various options for you to customize where your users are redirected to.
```
Comment (S3Prototype) [inline docs/account-portal/custom-redirects.mdx:8]:
```suggestion
When a user navigates to a Clerk sign up or sign in page via a link or button, Clerk will:
1. Persist the previous page's URL in a `redirect_url` query string
2. Navigate back to that page after the sign-up or sign-in is completed

This avoids interrupting the user's flow through your app. For example, a user selecting a sign in button on `example.com/foo` will navigate to `example.com/sign-in?redirect_url=example.com/foo`, then navigate back to `example.com/foo` upon completing the sign-in process.

The following sections describe the options Clerk provides for you to customize where your users are redirected to upon completing a sign-in or sign-up process.
```

Just breaks the text up and rewords it for easier readability

File: docs/account-portal/custom-redirects.mdx [hunk 2]
```diff
@@ -5,21 +5,35 @@ description: Customize where your users are redirected to after they sign in or
...
 
 ```sh filename=".env"
-CLERK_AFTER_SIGN_IN_URL=/dashboard
-CLERK_AFTER_SIGN_UP_URL=/onboarding
+CLERK_SIGN_IN_FALLBACK_REDIRECT_URL=/dashboard
+CLERK_SIGN_UP_FALLBACK_REDIRECT_URL=/onboarding
+```
+</CodeBlockTabs>
+
+If you'd like to force the redirect regardless of whether the user was on another page previously, you can use the "force" environment variable values instead as such:
```
Comment (S3Prototype) [inline docs/account-portal/custom-redirects.mdx:26]:
```suggestion
If you'd like to force the redirect regardless of whether the user was on another page previously, you can use the following `FORCE` environment variable values instead:
```

File: docs/account-portal/custom-redirects.mdx [hunk 3]
```diff
@@ -39,9 +53,18 @@ export const config = {
 };
 ```
 
-## `afterSignInUrl` and `afterSignUpUrl` props
+## Redirect URL props
+
+You can explicitly define the redirect paths after sign up or sign in by passing one of the following properties to the respective components:
```
Comment (S3Prototype) [inline docs/account-portal/custom-redirects.mdx:58]:
```suggestion
You can explicitly define the redirect paths after sign-up or sign-in by passing one of the following properties to the respective components:
```

**Comment by S3Prototype** [comment]
@jescalan In the interest of saving time, would it be okay for me to pull this PR locally and make changes directly? Then you can verify that it's still technically accurate, and Alexis can review before we merge? There are some structural changes I want to make that are not communicated well in PR comments

**Comment by S3Prototype** [comment]
I pushed my changes 👍

**Review by alexisintech — APPROVED** [review]

**Pull request merged**